dwave/optimization/src/array.cpp

```diff
@@ -38,8 +38,7 @@ bool SizeInfo::operator==(const SizeInfo& other) const {
     if (this->array_ptr != other.array_ptr) return false;
 
     return (this->multiplier == other.multiplier && this->offset == other.offset &&
-            this->min.value_or(0) == other.min.value_or(0) &&
-            this->max == other.max);
+            this->min.value_or(0) == other.min.value_or(0) && this->max == other.max);
```

Comment:
Please don't edit unrelated functions for formatting. It clutters the git history and without https://github.com/dwavesystems/dwave-optimization/issues/113 it's not even deterministic.